xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextExtension.xtend

```diff
@@ -73,14 +77,14 @@ class XtextExtension {
 
 @Accessors
 class Language implements Named {
-	val String name
-	String qualifiedName
-	String fileExtension
-	String setup
-	val GeneratorConfig generator
-	val debugger = new DebuggerConfig
-	val validator = new ValidatorConfig
-	Map<String, Object> preferences = newHashMap
+	@Input val String name
+	@Input String qualifiedName
+	@Input String fileExtension
+	@Input String setup
+	@Nested val GeneratorConfig generator
+	@Nested val debugger = new DebuggerConfig
+	@Nested val validator = new ValidatorConfig
+	@Input Map<String, Object> preferences = newHashMap
```

Comment:
We should make this a linkedhashmap, otherwise the order may change from VM to VM.